docs/_articles/en/einstein/einstein-troubleshoot.md

```diff
@@ -5,4 +5,33 @@ lang: en
 
 Generative AI uses large language models (LLMs) to generate the output. Because LLMs are trained by other organizations, the output isn’t always what you expect, and the code that’s generated isn’t perfect. If the output doesn’t meet your expectations, you can generate a new output by starting over. The generated output from the previous attempt isn’t saved, and the new output replaces it.
 
-To keep track of all known issues and potential workarounds, use the [Issues](https://github.com/forcedotcom/Einstein-GPT-for-Developers/issues) tab in this repo.
+To track all known issues and potential workarounds, use the [Issues](https://github.com/forcedotcom/Einstein-GPT-for-Developers/issues) tab in this repo.
+
+**Note:** If your company requires the use of a proxy, we recommend setting this through the Salesforce CLI, not through VS Code, in order to use the Einstein for Developers extension.
+
+## Troubleshooting Checklist
+Use this troubleshooting checklist to identify and eliminate common problems with using Einstein for Developers.
+
+### Installation Checklist
+
+1. Confirm that you have the [Salesforce Extension Pack](https://marketplace.visualstudio.com/items?itemName=salesforce.salesforcedx-vscode) or the [Salesforce Expanded Pack](https://marketplace.visualstudio.com/items?itemName=salesforce.salesforcedx-vscode-expanded) installed.
+2. Check that you are on VS Code version 1.82.0 or higher. 
+3. Check that your Apex Extension is version 59.8.0 or higher.
+4. Hover over {} in the status bar to view the status of the Apex Language Server and make sure it’s running.  
+5. Einstein for Developers needs Salesforce CLI version 59.13.0 or higher. Make sure your CLI version hasn’t fallen out of sync. See [Salesforce CLI](https://developer.salesforce.com/tools/salesforcecli). 
+
+### Development Environment Checklist
+
+1. Einstein for Developers only runs inside a Salesforce project. Confirm that you have a Salesforce DX project open. 
```

Comment:
```suggestion
1. Einstein for Developers only runs inside a Salesforce project. Confirm that you have a Salesforce DX project open by checking that you have an `sfdx-project.json` file stored in your workspace. More information on the Salesforce Project Configuration can be found in the Salesforce CLI docs [here](https://developer.salesforce.com/docs/atlas.en-us.sfdx_dev.meta/sfdx_dev/sfdx_dev_ws_config.htm).
```

Reply:
@klewis-sfdc does that seem more helpful? @sbudhirajadoc do you think we'd be ok to point to a link on the CLI docs page? I'm mostly making sure it's not changing underfoot and that it's not overkill here.